stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/App.kt

```diff
@@ -41,6 +44,11 @@ class App : Application() {
         }
     }
 
+    @OptIn(InternalStreamChatApi::class)
+    private fun initializeToggleService() {
+        ToggleService.init(applicationContext, mapOf(ToggleService.TOGGLE_KEY_OFFLINE to true))
```

Comment:
What if init toggle service with true flag in debug and false in release version?